manifest.json

```diff
@@ -1,30 +1,30 @@
 {
-  "serial": 2022031101,
-  "hash": "sJjqvFQYEv9EJYlSMxKOrcdBNntu2i6zj9itR8FwgGc=",
+  "serial": 2022031201,
+  "hash": "hSXJpydvavfyFyGni5oHEnK0oqGlzl1PtBqH5U12Ikg=",
   "signatures": [
     {
       "address": "13YqCei6dP2ibq2DAy81NmU8FYwwzo5HLKXFA3yM3Rvq9WxDfKF",
-      "signature": "AV8EdhyDaEVEcw091d2/2fySDOoCsudI2C2llM31CnPKYDIKHN4fDDkd1tYvJYYE/nR4BfuRB+zAaVTL6DDAAg=="
+      "signature": ""
     },
     {
       "address": "14egjs9cSxt4jnaZTYQE4NemQVXL6qtayqNNpy3VJPSS6AAcaUq",
-      "signature": "SD/+Ul6zu8OhOks5+bDgM/p4BiHM5dFAWF+L55AJvW/opMnhSfyufJSDRfnuiDaDhbXJrG4ApnrzbKjHxWA7Bg=="
+      "signature": ""
     },
     {
       "address": "13RKFFy5qmibyaS6N3Uw18UBDwGhhsXpk6jDn7SMenfDrQ6o6eh",
-      "signature": "wtwnlN1tefyrKpKbq7Uhp0e1uenYiAf1mizwvIWLs+UGMvTznDf7C/pD5ec/ob2k3mGa46gdJEvtTPI8Tb/yCQ=="
+      "signature": ""
     },
     {
       "address": "13t59Q3oihn5YnhmW4bJARbxrK9TWaXgn2SUXZYpZYBScjKycYZ",
       "signature": ""
```

Comment:
```suggestion
      "signature": "gYXrBiaskqurUXmTIWK0xnKD14czXGkkxRHHgjVSjXePJQoo4uUbxnlaR3pZfGNFRN77aYaQusjmPDwMu+ezBg=="
```

Reply:
This appears to be an incorrect signature.